spec/spec_helper.rb

```diff
@@ -33,7 +34,7 @@
   config.include SurveyorAPIHelpers
   config.include WaitForAjax
 
-  config.treat_symbols_as_metadata_keys_with_true_values = true
+  # config.treat_symbols_as_metadata_keys_with_true_values = true
```

Comment:
I forget what this was originally about, so probably worth double checking (and just removing if not needed)

Reply:
I think we might want it https://relishapp.com/rspec/rspec-core/v/2-99/docs/configuration/treat-symbols-as-metadata-keys-with-true-values
Although it doesn't seem to make a difference in the spec runs. I have a feeling it was added to handle the `DatabaseCleaner.strategy` configuration below